android/assets/jsons/Civ V - Vanilla/Terrains.json

```diff
@@ -83,9 +83,9 @@
 	{
 		"name": "Mountain",
 		"type": "Land",
-		"movementCost": 1,
 		"impassable": true,
 		"defenceBonus": 0.25,
+		"damagePerTurn": 50,
```

Comment:
As this an 'ability' only a handful (namely one) of terrains have, I'd personally made this a unique instead of another field

Reply:
Alright but I still think it should be stored as an Int in the actual class since it's accessed often.

Reply:
You can always add a `terrain.setTransients()` function called from `tile.setTransients()` or `map.setTransients()` depending on what already exists